app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -223,6 +238,7 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
                 logger.warn(s"Could not write to keystore: $e")
                 SeeOther(routes.StrideController.getErrorPage().url)
             }, { _ ⇒
+              auditor.sendEvent(ManualAccountCreationSelected(nSIUserInfo.nino, request.path, operatorDetails), nSIUserInfo.nino)
```

Comment:
why are we creating the event here but not right after `AccountCreated` which the original code was doing ? 
also what happens if account creation fails after this event ?  then we end up with an event but no account created

Reply:
hmm ok, but what if the account creation fails in the backed ?  then there will be a stale event hanging around  ? also if operator refreshes the page then won't it trigger another event for the same ?

Reply:
in that scenario that `ManualAccountCreationSelected` event will still be there, but it still serves as a record of what a particular call handler has done. If a call handler refreshes the page, they won't see a duplicate event - the only time you'd get a duplicate event is if the call handler chooses the manual account option and presses continue, goes to the next page and then presses back to the previous page and selects manual account creation again. Duplicate events aren't importatn however - what's important is having a record of a call handler choosing to do manual account creation